bin/dup-pr

```diff
@@ -0,0 +1,271 @@
+#!/bin/bash
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Copyright 2019 Joyent, Inc.
+#
+
+#
+# Call this script to open a new PR to duplicate one created by
+# 'joyent-automation'. Once pushed, that new PR will have the 'Author:'
+# field set to the user who ran this script, rather than 'Joyent Automation'
+#
+# This will do the following:
+# - Use 'hub api' to get the body, submitter and creation date of a PR
+# - If the PR is owned by 'joyent-automation', get the PR body and collect
+#   comments and review comments made to the PR after its creation
+# - rename the local and remote branches, appending '-dup-pr' to allow us
+#   to create a new PR
+# - Use 'hub api' to create that identical PR using the body, referring to the
+#   original PR
+# - re-assign reviewers to the new PR
+#
+# Prerequisites:
+# - You have "json" and "hub" installed and on your PATH.
+#
+# Limitations:
+# -
+#
+# Usage (prompts for parameters):
+#   bash -c "$(curl -ksSL https://raw.githubusercontent.com/joyent/gerrit-migration/master/bin/dup-pr)"
+#
+# or:
+#   curl -ksSL -O https://raw.githubusercontent.com/joyent/gerrit-migration/master/bin/dup-pr
+#   chmod +x ./dup-pr
+#   ./dup-pr [-C git repo, to lookup 'origin'] PR-NUM ...
```

Comment:
Not worth changing now, but that `-C arg` could have been (a) a positional arg (it is required) and (b) just the name of the repo, e.g. "foo", which would imply origin "git@github.com/joyent/foo.git".

Reply:
Yep, that's reasonable, though I'm not convinced about the positional args argument (this may be an "emacs vs. vi conversation", and one discussion over beer instead :)